### DISCUSSION TITLE
Hello, can you add a function: automatically upload network disk after every  downloads (via rclone)？

### DISCUSSION THREAD

**Discussion by Mpk2022** [Dineshkarthik/telegram_media_downloader#157] [Ideas]
I can upload via "rclone move -v ./telegram_media_downloader/video/rclone_name:/ --transfers=1", I hope you can write it into the code. I found an example:https://github.com/snow922841/telegram_channel_downloader

**Comment by Dineshkarthik** [comment]
Hi, upload is out of scope for this application. Let's see if more user wants this then I can add it to the roadmap.

**Reply by cj1354421348** [reply]
Hope to increase support for rclone.
